server/session/session.proto

```diff
@@ -26,10 +30,17 @@ message SessionResponse {
 service SessionService {
 
   // Create a new JWT for authentication.
-  rpc Create(SessionRequest) returns (SessionResponse) {
+  rpc Create(SessionCreateRequest) returns (SessionResponse) {
     option (google.api.http) = {
-      post: "/api/v1/sessions/create"
+      post: "/api/v1/session/create"
```

Comment:
I think we should lose the /create and the /delete endpoints. This is implied by the HTTP method

Reply:
@jessesuen I'm not sure what's being suggested.  Currently I'm under the impression that the `Create` method, regardless of name, serves two functions:

1. Create a cookie for the Web client.
2. Create a token and return it (for use by the CLI).

The `Delete` method, meanwhile, simply sets the cookie again, but to a blank value, and doesn't do any username/password checking.

If we combine `Create` and `Delete` into a single Go method, switching on HTTP method, there isn't a unified SessionRequest type since username and password are required for login but useless for logout.  Unless I'm mistaken, we also won't have an HTTP method on which to switch when logging in with the CLI.

I suspect there's some feature of gRPC that I'm not understanding fully, hence the above confusion.

Reply:
There would still be two gRPC methods/implementations. I'm just saying that the URI that maps to those two methods should just become:
```
post: "/api/v1/session"
```
and 
```
delete: "/api/v1/session"
```
for create session and delete session, respectively.

Reply:
@jessesuen ah, I understand now!   Changes made.